internal/pkg/api/handleUpload.go

```diff
@@ -209,6 +209,7 @@ func (ut *UploadT) handleUploadChunk(zlog *zerolog.Logger, w http.ResponseWriter
 		return uploader.ErrHashMismatch
 	}
 
+	w.WriteHeader(http.StatusOK)
```

Comment:
Would this be needed in other handlers here too?

Reply:
By default Go takes care of it if no call has been made to `WriteHeader` before `Write`:
> 	// If WriteHeader is not called explicitly, the first call to Write
	// will trigger an implicit WriteHeader(http.StatusOK).

Reply:
Is this needed because `w.Write(...)` is not called? I would expect this is required here because there is no call to `w.Write(...)`.

Reply:
currently nothing gets written for a successful chunk upload so we need to explicitly write the header